invoke/runners.py

```diff
@@ -476,7 +490,7 @@ def start(self, command):
             )
 
     def default_encoding(self):
-        return locale.getpreferredencoding(False)
+        return locale.getpreferredencoding()
```

Comment:
@pfmoore I've seen some bug reports around this `locale.getpreferredencoding` thing, but I cannot think of any other way of getting a proper encoding. I would also prefer not to touch a global state... We can try using `locale.getdefaultlocale()`, but it reports `(None, None)` when I set `LC_ALL=C`, so we will probably need to combine these two for Python 2.x:

Linux UTF-8 locale:

```
$ python -c 'import locale; print(locale.getdefaultlocale())'
('en_US', 'UTF-8')
$ python -c 'import locale; print(locale.getpreferredencoding())'
UTF-8
$ python -c 'import locale; print(locale.getpreferredencoding(False))'
ANSI_X3.4-1968
```

Linux C (ASCII) locale:

```
$ python -c 'import locale; print(locale.getdefaultlocale())'
(None, None)
$ python -c 'import locale; print(locale.getpreferredencoding())'
ANSI_X3.4-1968
$ python -c 'import locale; print(locale.getpreferredencoding(False))'
ANSI_X3.4-1968
```

Mac OS:

```
salford:~ frol$ python -c 'import locale; print(locale.getdefaultlocale())'
('en_US', 'UTF-8')
salford:~ frol$ python -c 'import locale; print(locale.getpreferredencoding())'
UTF-8
salford:~ frol$ python -c 'import locale; print(locale.getpreferredencoding(False))'
US-ASCII
```

Windows cp1252 locale (bat and Cygwin sh):

```
$ python -c 'import locale; print(locale.getdefaultlocale())'
('en_US', 'cp1252')
$ python -c 'import locale; print(locale.getpreferredencoding())'
cp1252
$ python -c 'import locale; print(locale.getpreferredencoding(False))'
cp1252
```

I'll fix my PR based on this info.